runtime/vm/jvminit.c

```diff
@@ -1610,7 +1610,20 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				goto _memParseError;
 			}
 
-			/* Set entitled CPUs as early as possible, i.e. as soon as PORT_LIBRARY_GUARANTEED */
+			/* Set active and entitled CPUs as early as possible, i.e. as soon as PORT_LIBRARY_GUARANTEED */
+			if ((argIndex = FIND_AND_CONSUME_ARG(STARTSWITH_MATCH, VMOPT_XXACTIVEPROCESSORCOUNT_EQUALS, NULL)) >= 0) {
```

Comment:
Can you split this into two lines?  The first assigning to argIndex and the second doing the >= check?

Code is read more often than it's written.  Having each line do one thing helps the reader to easily and quickly see what's happening in the code.